common/script_values/wc_men_at_arms_values.txt

```diff
@@ -0,0 +1,130 @@
+﻿######################
+# Men-at-Arms Values #
+######################
+
+#########
+# Costs #
+#########
+
+@recr_to_maint_factor = 0.004
+@high_maint_mult = 3
+
+### Base Cost Values
+@massive_infantry_recruitment_cost = 150
+@massive_infantry_low_maint_cost = @[massive_infantry_recruitment_cost * recr_to_maint_factor]
+@massive_infantry_high_maint_cost = @[massive_infantry_low_maint_cost * high_maint_mult]
+
+@caster_recruitment_cost = 100
+@caster_low_maint_cost = @[caster_recruitment_cost * recr_to_maint_factor]
+@caster_high_maint_cost = @[caster_low_maint_cost * high_maint_mult]
+
+@light_aerial_recruitment_cost = 100
+@light_aerial_low_maint_cost = @[light_aerial_recruitment_cost * recr_to_maint_factor]
+@light_aerial_high_maint_cost = @[light_aerial_low_maint_cost * high_maint_mult]
+
+@massive_aerial_recruitment_cost = 250
+@massive_aerial_low_maint_cost = @[massive_aerial_recruitment_cost * recr_to_maint_factor]
+@massive_aerial_high_maint_cost = @[massive_aerial_low_maint_cost * high_maint_mult]
+
+### Base MAA
+massive_infantry_recruitment_cost = @[massive_infantry_recruitment_cost]
+massive_infantry_low_maint_cost = @[massive_infantry_low_maint_cost]
+massive_infantry_high_maint_cost = @[massive_infantry_high_maint_cost]
+
+caster_recruitment_cost = @[caster_recruitment_cost]
+caster_low_maint_cost = @[caster_low_maint_cost]
+caster_high_maint_cost = @[caster_high_maint_cost]
+
+light_aerial_recruitment_cost = @[light_aerial_recruitment_cost]
+light_aerial_low_maint_cost = @[light_aerial_low_maint_cost]
+light_aerial_high_maint_cost = @[light_aerial_high_maint_cost]
+
+massive_aerial_recruitment_cost = @[massive_aerial_recruitment_cost]
+massive_aerial_low_maint_cost = @[massive_aerial_low_maint_cost]
+massive_aerial_high_maint_cost = @[massive_aerial_high_maint_cost]
+
+### Cultural MAA
+#Add custom costs for each cultural maa here
+### Regional MAA
+#Add custom costs for each regional maa here
+
+##############
+# AI Weights #
+##############
+#Which cultures like what type of unit
+# Warcraft
+culture_ai_weight_caster = {
+	value = 168.7
+	
+	if = { # Cultures that favours casters
+		limit = {
+			OR = {
+				culture_group = culture_group:elf_group
+				culture = culture:dalaranian
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike casters
+		limit = {
+			taur_culture_trigger = yes
+			winged_culture_trigger = yes
+		}
```

Comment:
Both triggers must be true. The question, how one character can have 2 cultures at the same time?

Reply:
Can I use an OR statement here?

Reply:
Yes.